geometry_msgs/Pose tl_vertex
geometry_msgs/Pose tr_vertex
geometry_msgs/Pose br_vertex
geometry_msgs/Pose bl_vertex
int32 cell_type
